Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+FROM rust:slim-bullseye
```

Comment:
This causes the large size. The rust image was the only one that worked out of python, ubuntu, and debian,